### ISSUE TITLE
Accessibility - First Run carousel tabs are not labeled 

### ISSUE THREAD

**Issue by ekager** [mozilla-mobile/focus-android#2781]
The carousel's breadcrumbs(?) are not labeled. The current one will say "selected", but that's it. How should they be exposed? Not sure. Google's "Docs" app has a ftu too, and the dots are simply not accessible to the screen reader. I would label them "screen 1", "screen 2", "screen 3", "screen 4". and then you would hear "screen 2, selected", when in the second screen and you can get to the other ones as well.

<img alt="Carousel dots are unlabeled" src="https://user-images.githubusercontent.com/5764/39452700-fdd115d0-4c87-11e8-9346-797fb0ab0224.png" width="256">
